### PR TITLE
Improve Ballerina-file API doc

### PR DESCRIPTION
**Pull request by niveathika**
## Purpose
Improve Ballerina-file API doc

## Check List 
- [x] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: stdlib/file/src/main/ballerina/src/file/Module.md [hunk 1]
```diff
@@ -1,52 +1,10 @@
-## Module overview
-
 This module contains functions to perform file system based operations such as create, delete, rename the
 file/directory and retrieve metadata of the file.
```
Comment (praneesha) [inline stdlib/file/src/main/ballerina/src/file/Module.md:2]:
```suggestion
file/directory, and retrieve metadata of the file.
```

File: stdlib/file/src/main/ballerina/src/file/Module.md [hunk 2]
```diff
@@ -1,52 +1,10 @@
...
-
-    // Rename(Move) the file or directory to the new path.
-    error? results = file:rename("/A/B/C", "/A/B/D");
-
-    // Get default directory use for temporary files.
-    string results = file:tempDir();
-
-    // Create a file in given file path.
-    string | error results = file:createFile("bar.txt");
+For an example on the usage of file operations, see the [File Example](https://ballerina.io/v1-2/learn/by-example/file.html).
```
Comment (praneesha) [inline stdlib/file/src/main/ballerina/src/file/Module.md:4]:
```suggestion
For an example on the usage of the operations, see the [File Example](https://ballerina.io/learn/by-example/file.html).
```

File: stdlib/file/src/main/ballerina/src/file/Module.md [hunk 3]
```diff
@@ -1,52 +1,10 @@
...
 
-    // Get the list of files in the directory.
-    file:FileInfo[] | error results = file:readDir("foo/bar");
+The `Directory Listener` is used to listen to a directory in the local file system. It notifies when new files are created in the directory or when the existing files are deleted or modified.
 
-    // Copy the file or directory to the new path.
-    error? results = file:copy("/A/B/C", "/A/B/D", true);
-}
-```
+For an example on the usage of `Directory Listener` see the [Directory Listener Example](https://ballerina.io/v1-2/learn/by-example/directory-listener.html).
```
Comment (praneesha) [inline stdlib/file/src/main/ballerina/src/file/Module.md:10]:
```suggestion
For an example on the usage of the operations, see the [Directory Listener Example](https://ballerina.io/learn/by-example/directory-listener.html).
```

File: stdlib/file/src/main/ballerina/src/file/file.bal [hunk 4]
```diff
@@ -17,8 +17,11 @@
 import ballerina/java;
 
 # Returns the current working directory.
-#
-# + return - Current working directory or an empty string if the current working directory cannot be determined
+# ```ballerina
+# string dirPath = file:getCurrentDirectory();
+# ```
+# 
+# + return - Current working directory, or else an empty string if the current working directory cannot be determined
```
Comment (praneesha) [inline stdlib/file/src/main/ballerina/src/file/file.bal:24]:
```suggestion
# + return - Current working directory or else an empty string if the current working directory cannot be determined
```

File: stdlib/file/src/main/ballerina/src/file/file.bal [hunk 5]
```diff
@@ -29,9 +32,12 @@ function externGetCurrentDirectory() returns handle = @java:Method {
 } external;
 
 # Reports whether file or directory exists for the given the path.
```
Comment (praneesha) [inline stdlib/file/src/main/ballerina/src/file/file.bal:34]:
```suggestion
# Reports whether the file or directory exists in the given the path.
```

File: stdlib/file/src/main/ballerina/src/file/file.bal [hunk 6]
```diff
@@ -29,9 +32,12 @@ function externGetCurrentDirectory() returns handle = @java:Method {
...
+# ```ballerina
+# boolean result = file:exists("foo/bar.txt");
+# ```
 #
-# + path - String value of file path.
-# + return - True if path is absolute, else false
+# + path - String value of file path
```
Comment (praneesha) [inline stdlib/file/src/main/ballerina/src/file/file.bal:39]:
```suggestion
# + path - String value of the file path
```

File: stdlib/file/src/main/ballerina/src/file/file.bal [hunk 7]
```diff
@@ -29,9 +32,12 @@ function externGetCurrentDirectory() returns handle = @java:Method {
...
+# + return - True if path is absolute, or else false
```
Comment (praneesha) [inline stdlib/file/src/main/ballerina/src/file/file.bal:40]:
```suggestion
# + return - True if the path is absolute or else false
```

File: stdlib/file/src/main/ballerina/src/file/file.bal [hunk 8]
```diff
@@ -43,10 +49,13 @@ function externExists(handle path) returns boolean = @java:Method {
 
 # Creates a new directory with the specified file name.
 # If parentDirs flag is true, Creates a directory in specified path with any necessary parents.
```
Comment (praneesha) [inline stdlib/file/src/main/ballerina/src/file/file.bal:51]:
```suggestion
# If the `parentDirs` flag is true, it creates a directory in the specified path with any necessary parents.
```

File: stdlib/file/src/main/ballerina/src/file/file.bal [hunk 9]
```diff
@@ -43,10 +49,13 @@ function externExists(handle path) returns boolean = @java:Method {
...
+# ```ballerina
+# string | error results = file:createDir("foo/bar");
+# ```
 #
-# + dir - directory name.
-# + parentDirs - Indicates whether the `createDir` should create non-existing parent directories.
-# + return - Returns absolute path value of the created directory or an `Error` if failed
+# + dir - Directory name
+# + parentDirs - Indicates whether the `createDir` should create non-existing parent directories
+# + return - Absolute path value of the created directory, or else an `Error` if failed
```
Comment (praneesha) [inline stdlib/file/src/main/ballerina/src/file/file.bal:58]:
```suggestion
# + return - Absolute path value of the created directory or else an `Error` if failed
```
Need to specify the error.

File: stdlib/file/src/main/ballerina/src/file/file.bal [hunk 10]
```diff
@@ -63,10 +72,13 @@ function externCreateDir(handle dir, boolean parentDirs) returns handle|Error =
 
 # Removes the specified file or directory.
 # If recursive flag is true, Removes the path and any children it contains.
```
Comment (praneesha) [inline stdlib/file/src/main/ballerina/src/file/file.bal:74]:
```suggestion
# If the recursive flag is true, it removes the path and any children it contains.
```

File: stdlib/file/src/main/ballerina/src/file/file.bal [hunk 11]
```diff
@@ -63,10 +72,13 @@ function externCreateDir(handle dir, boolean parentDirs) returns handle|Error =
...
+# ```ballerina
+# file:Error? results = file:remove("foo/bar.txt");
+# ```
 #
-# + path - String value of file/directory path.
-# + recursive - Indicates whether the `remove` should recursively remove all the file inside the given directory.
-# + return - Returns an `Error` if failed to remove.
+# + path - String value of file/directory path
```
Comment (praneesha) [inline stdlib/file/src/main/ballerina/src/file/file.bal:79]:
```suggestion
# + path - String value of the file/directory path
```

File: stdlib/file/src/main/ballerina/src/file/file.bal [hunk 12]
```diff
@@ -63,10 +72,13 @@ function externCreateDir(handle dir, boolean parentDirs) returns handle|Error =
...
+# + recursive - Indicates whether the `remove` should recursively remove all the file inside the given directory
```
Comment (praneesha) [inline stdlib/file/src/main/ballerina/src/file/file.bal:80]:
```suggestion
# + recursive - Indicates whether the `remove` should recursively remove all the files inside the given directory
```

File: stdlib/file/src/main/ballerina/src/file/file.bal [hunk 13]
```diff
@@ -63,10 +72,13 @@ function externCreateDir(handle dir, boolean parentDirs) returns handle|Error =
...
+# + return - An `Error` if failed to remove
```
Comment (praneesha) [inline stdlib/file/src/main/ballerina/src/file/file.bal:81]:
```suggestion
# + return - An `Error` if failed to remove
```
Do we need to specify the error?

File: stdlib/file/src/main/ballerina/src/file/file.bal [hunk 14]
```diff
@@ -78,10 +90,13 @@ function externRemove(handle path, boolean recursive) returns Error? = @java:Met
 
 # Renames(Moves) old path to new path.
```
Comment (praneesha) [inline stdlib/file/src/main/ballerina/src/file/file.bal:91]:
```suggestion
# Renames(Moves) the old path with the new path.
```

File: stdlib/file/src/main/ballerina/src/file/file.bal [hunk 15]
```diff
@@ -78,10 +90,13 @@ function externRemove(handle path, boolean recursive) returns Error? = @java:Met
...
 # If new path already exists and it is not a directory, this replaces the file.
```
Comment (praneesha) [inline stdlib/file/src/main/ballerina/src/file/file.bal:92]:
```suggestion
# If the new path already exists and it is not a directory, this replaces the file.
```

File: stdlib/file/src/main/ballerina/src/file/file.bal [hunk 16]
```diff
@@ -78,10 +90,13 @@ function externRemove(handle path, boolean recursive) returns Error? = @java:Met
...
+# ```ballerina
+# file:error? results = file:rename("/A/B/C", "/A/B/D");
+# ```
 #
-# + oldPath - String value of old file path.
-# + newPath - String value of new file path.
-# + return - Returns an `Error` if failed to rename.
+# + oldPath - String value of old file path
```
Comment (praneesha) [inline stdlib/file/src/main/ballerina/src/file/file.bal:97]:
```suggestion
# + oldPath - String value of the old file path
```

File: stdlib/file/src/main/ballerina/src/file/file.bal [hunk 17]
```diff
@@ -78,10 +90,13 @@ function externRemove(handle path, boolean recursive) returns Error? = @java:Met
...
+# + newPath - String value of new file path
```
Comment (praneesha) [inline stdlib/file/src/main/ballerina/src/file/file.bal:98]:
```suggestion
# + newPath - String value of the new file path
```

File: stdlib/file/src/main/ballerina/src/file/file.bal [hunk 18]
```diff
@@ -78,10 +90,13 @@ function externRemove(handle path, boolean recursive) returns Error? = @java:Met
...
+# + return - An `Error` if failed to rename
```
Comment (praneesha) [inline stdlib/file/src/main/ballerina/src/file/file.bal:99]:
```suggestion
# + return - An `Error` if failed to rename
```
Do we need to specify the error?

File: stdlib/file/src/main/ballerina/src/file/file.bal [hunk 19]
```diff
@@ -105,9 +123,12 @@ function externTempDir() returns handle = @java:Method {
 
 # Creates a file in specified file path.
```
Comment (praneesha) [inline stdlib/file/src/main/ballerina/src/file/file.bal:124]:
```suggestion
# Creates a file in the specified file path.
```

File: stdlib/file/src/main/ballerina/src/file/file.bal [hunk 20]
```diff
@@ -105,9 +123,12 @@ function externTempDir() returns handle = @java:Method {
...
 # Truncates if file already exists in the given path.
```
Comment (praneesha) [inline stdlib/file/src/main/ballerina/src/file/file.bal:125]:
```suggestion
# Truncates if the file already exists in the given path.
```

File: stdlib/file/src/main/ballerina/src/file/file.bal [hunk 21]
```diff
@@ -105,9 +123,12 @@ function externTempDir() returns handle = @java:Method {
...
+# ```ballerina
+# string | error results = file:createFile("bar.txt");
+# ```
 #
-# + path - String value of file path.
-# + return - Returns absolute path value of the created file or an `Error` if failed
+# + path - String value of file path
```
Comment (praneesha) [inline stdlib/file/src/main/ballerina/src/file/file.bal:130]:
```suggestion
# + path - String value of the file path
```

File: stdlib/file/src/main/ballerina/src/file/file.bal [hunk 22]
```diff
@@ -105,9 +123,12 @@ function externTempDir() returns handle = @java:Method {
...
+# + return - Absolute path value of the created file, or else an `Error` if failed
```
Comment (praneesha) [inline stdlib/file/src/main/ballerina/src/file/file.bal:131]:
```suggestion
# + return - Absolute path value of the created file or else an `Error` if failed
```
Do we need to specify the error?

File: stdlib/file/src/main/ballerina/src/file/file.bal [hunk 23]
```diff
@@ -123,9 +144,12 @@ function externCreateFile(handle path) returns handle|Error = @java:Method {
 } external;
 
 # Returns metadata information of the file specified in file path.
```
Comment (praneesha) [inline stdlib/file/src/main/ballerina/src/file/file.bal:146]:
```suggestion
# Returns the metadata information of the file specified in the file path.
```

File: stdlib/file/src/main/ballerina/src/file/file.bal [hunk 24]
```diff
@@ -123,9 +144,12 @@ function externCreateFile(handle path) returns handle|Error = @java:Method {
...
+# ```ballerina
+# file:FileInfo | error result = file:getFileInfo("foo/bar.txt");
+# ```
 #
 # + path - String value of the file path.
-# + return - Returns FileInfo instance with file metadata or an `Error`
+# + return - FileInfo instance with file metadata, or else an `Error`
```
Comment (praneesha) [inline stdlib/file/src/main/ballerina/src/file/file.bal:152]:
```suggestion
# + return - The `FileInfo` instance with the file metadata or else an `Error`
```
Do we need to specify the error?

File: stdlib/file/src/main/ballerina/src/file/file.bal [hunk 25]
```diff
@@ -135,11 +159,14 @@ function externGetFileInfo(handle path) returns FileInfo|Error = @java:Method {
     name: "getFileInfo"
 } external;
 
-# Reads the directory and returns a list of files and directories # inside the specified directory
+# Reads the directory and returns a list of files and directories # inside the specified directory.
```
Comment (praneesha) [inline stdlib/file/src/main/ballerina/src/file/file.bal:162]:
```suggestion
# Reads the directory and returns a list of files and directories 
# inside the specified directory.
```

File: stdlib/file/src/main/ballerina/src/file/file.bal [hunk 26]
```diff
@@ -135,11 +159,14 @@ function externGetFileInfo(handle path) returns FileInfo|Error = @java:Method {
...
+# ```ballerina
+# file:FileInfo[] | error results = file:readDir("foo/bar");
+# ```
 #
 # + path - String value of directory path.
```
Comment (praneesha) [inline stdlib/file/src/main/ballerina/src/file/file.bal:167]:
```suggestion
# + path - String value of the directory path.
```

File: stdlib/file/src/main/ballerina/src/file/file.bal [hunk 27]
```diff
@@ -135,11 +159,14 @@ function externGetFileInfo(handle path) returns FileInfo|Error = @java:Method {
...
-# + maxDepth - The maximum number of directory levels to visit. -1 to indicate that all levels should be visited.
-# + return - Returns FileInfo array or an `Error` if there is an error while changing the mode.
+# + maxDepth - The maximum number of directory levels to visit. -1 to indicate that all levels should be visited
+# + return - FileInfo array, or else an `Error` if there is an error while changing the mode.
```
Comment (praneesha) [inline stdlib/file/src/main/ballerina/src/file/file.bal:169]:
```suggestion
# + return - The `FileInfo` array or else an `Error` if there is an error while changing the mode.
```
Do we need to specify the error?

File: stdlib/file/src/main/ballerina/src/file/file.bal [hunk 28]
```diff
@@ -151,11 +178,14 @@ function externReadDir(handle path, int maxDepth) returns FileInfo[]|Error = @ja
 
 # Copy file/directory in old path to new path.
```
Comment (praneesha) [inline stdlib/file/src/main/ballerina/src/file/file.bal:179]:
```suggestion
# Copy the file/directory in the old path to the new path.
```

File: stdlib/file/src/main/ballerina/src/file/file.bal [hunk 29]
```diff
@@ -151,11 +178,14 @@ function externReadDir(handle path, int maxDepth) returns FileInfo[]|Error = @ja
...
 # If new path already exists, this replaces the file.
```
Comment (praneesha) [inline stdlib/file/src/main/ballerina/src/file/file.bal:180]:
```suggestion
# If a file already exists in the new path, this replaces that file.
```

File: stdlib/file/src/main/ballerina/src/file/file.bal [hunk 30]
```diff
@@ -151,11 +178,14 @@ function externReadDir(handle path, int maxDepth) returns FileInfo[]|Error = @ja
...
+# ```ballerina
+# file:Error? results = file:copy("/A/B/C", "/A/B/D", true);
+# ```
 #
-# + sourcePath - String value of old file path.
-# + destinationPath - String value of new file path.
-# + replaceExisting - Flag to allow replace if file already exists in destination path.
-# + return - Returns an `Error` if failed to rename.
+# + sourcePath - String value of old file path
```
Comment (praneesha) [inline stdlib/file/src/main/ballerina/src/file/file.bal:185]:
```suggestion
# + sourcePath - String value of the old file path
```

File: stdlib/file/src/main/ballerina/src/file/file.bal [hunk 31]
```diff
@@ -151,11 +178,14 @@ function externReadDir(handle path, int maxDepth) returns FileInfo[]|Error = @ja
...
+# + destinationPath - String value of new file path
```
Comment (praneesha) [inline stdlib/file/src/main/ballerina/src/file/file.bal:186]:
```suggestion
# + destinationPath - String value of the new file path
```

File: stdlib/file/src/main/ballerina/src/file/file.bal [hunk 32]
```diff
@@ -151,11 +178,14 @@ function externReadDir(handle path, int maxDepth) returns FileInfo[]|Error = @ja
...
+# + replaceExisting - Flag to allow replace if file already exists in destination path
```
Comment (praneesha) [inline stdlib/file/src/main/ballerina/src/file/file.bal:187]:
```suggestion
# + replaceExisting - Flag to replace if the file already exists in the destination path
```

File: stdlib/file/src/main/ballerina/src/file/file.bal [hunk 33]
```diff
@@ -151,11 +178,14 @@ function externReadDir(handle path, int maxDepth) returns FileInfo[]|Error = @ja
...
+# + return - An `Error` if failed to rename
```
Comment (praneesha) [inline stdlib/file/src/main/ballerina/src/file/file.bal:188]:
```suggestion
# + return - An `Error` if failed to rename
```
Do we need to specify the error?

File: stdlib/file/src/main/ballerina/src/file/file_info.bal [hunk 34]
```diff
@@ -57,21 +57,21 @@ public type FileInfo object {
 
     # Returns the last modified time of the file.
```
Comment (praneesha) [inline stdlib/file/src/main/ballerina/src/file/file_info.bal:58]:
```suggestion
    # Returns the last-modified time of the file.
```

File: stdlib/file/src/main/ballerina/src/file/file_info.bal [hunk 35]
```diff
@@ -57,21 +57,21 @@ public type FileInfo object {
...
     #
-    # + return - Last modified time of the file.
+    # + return - Last modified time of the file
```
Comment (praneesha) [inline stdlib/file/src/main/ballerina/src/file/file_info.bal:60]:
```suggestion
    # + return - Last-modified time of the file
```

File: stdlib/file/src/main/ballerina/src/file/service_endpoint.bal [hunk 36]
```diff
@@ -20,13 +20,16 @@ import ballerina/java;
 ///////////////////////////////////
 /// Direcotry Listener Endpoint ///
 ///////////////////////////////////
+
 # Represents directory listener endpoint where used to listen to a directory in the local file system.
```
Comment (praneesha) [inline stdlib/file/src/main/ballerina/src/file/service_endpoint.bal:24]:
```suggestion
# Represents the directory listener endpoint, which is used to listen to a directory in the local file system.
```

File: stdlib/file/src/main/ballerina/src/file/service_endpoint.bal [hunk 37]
```diff
@@ -20,13 +20,16 @@ import ballerina/java;
...
-#
 public type Listener object {
     private ListenerConfig config;
 
     *lang:Listener;
 
+    # Creates a new Directory listener.
+    # 
+    # + listenerConfig - ListenerConfig record with directory details
```
Comment (praneesha) [inline stdlib/file/src/main/ballerina/src/file/service_endpoint.bal:32]:
```suggestion
    # + listenerConfig - The `ListenerConfig` record with the directory details 
```

File: stdlib/file/src/main/ballerina/src/file/service_endpoint.bal [hunk 38]
```diff
@@ -35,22 +38,40 @@ public type Listener object {
         }
     }
 
+    # Starts the `file:Listener`.
+    # 
+    # + return - (), or else error upon failure to start the listener
```
Comment (praneesha) [inline stdlib/file/src/main/ballerina/src/file/service_endpoint.bal:43]:
```suggestion
    # + return - () or else error upon failure to start the listener 
```
Do we need to specify the error?

File: stdlib/file/src/main/ballerina/src/file/service_endpoint.bal [hunk 39]
```diff
@@ -35,22 +38,40 @@ public type Listener object {
...
 
+    # Starts the `file:Listener`.
+    # 
+    # + return - (), or else error upon failure to start the listener 
     public function __start() returns error? {
         return start(self);
     }
 
-    public function __gracefulStop() returns error? {
+    # Gracefully stops the `file:Listener`.
```
Comment (praneesha) [inline stdlib/file/src/main/ballerina/src/file/service_endpoint.bal:48]:
```suggestion
    # Stops the `file:Listener` gracefully.
```

File: stdlib/file/src/main/ballerina/src/file/service_endpoint.bal [hunk 40]
```diff
@@ -35,22 +38,40 @@ public type Listener object {
...
+    # 
+    # + return - (), or else error upon failure to stop the listener
```
Comment (praneesha) [inline stdlib/file/src/main/ballerina/src/file/service_endpoint.bal:50]:
```suggestion
    # + return - () or else error upon failure to stop the listener 
```
Do we need to specify the error?

File: stdlib/file/src/main/ballerina/src/file/service_endpoint.bal [hunk 41]
```diff
@@ -35,22 +38,40 @@ public type Listener object {
...
 
-    public function __gracefulStop() returns error? {
+    # Gracefully stops the `file:Listener`.
+    # 
+    # + return - (), or else error upon failure to stop the listener 
+        public function __gracefulStop() returns error? {
         return ();
     }
 
+    # Forecefully stops the `file:Listener`.
```
Comment (praneesha) [inline stdlib/file/src/main/ballerina/src/file/service_endpoint.bal:55]:
```suggestion
    # Stops the `file:Listener` forcefully.
```

File: stdlib/file/src/main/ballerina/src/file/service_endpoint.bal [hunk 42]
```diff
@@ -35,22 +38,40 @@ public type Listener object {
...
+    # 
+    # + return - (), or else error upon failure to stop the listener
```
Comment (praneesha) [inline stdlib/file/src/main/ballerina/src/file/service_endpoint.bal:57]:
```suggestion
    # + return - () or else error upon failure to stop the listener 
```
Do we need to specify the error?

File: stdlib/file/src/main/ballerina/src/file/service_endpoint.bal [hunk 43]
```diff
@@ -35,22 +38,40 @@ public type Listener object {
...
     }
 
+    # Forecefully stops the `file:Listener`.
+    # 
+    # + return - (), or else error upon failure to stop the listener 
     public function __immediateStop() returns error? {
         return ();
     }
 
+    # Binds a service to `file:Listener`.
```
Comment (praneesha) [inline stdlib/file/src/main/ballerina/src/file/service_endpoint.bal:62]:
```suggestion
    # Binds a service to the `file:Listener`.
```

File: stdlib/file/src/main/ballerina/src/file/service_endpoint.bal [hunk 44]
```diff
@@ -35,22 +38,40 @@ public type Listener object {
...
+    # 
+    # + s - Type descriptor of the service
+    # + name - Name of the service
+    # + return - (), or else error upon failure to attach to the service
```
Comment (praneesha) [inline stdlib/file/src/main/ballerina/src/file/service_endpoint.bal:66]:
```suggestion
    # + return - () or else error upon failure to attach to the service
```
Do we need to specify the error?

File: stdlib/file/src/main/ballerina/src/file/service_endpoint.bal [hunk 45]
```diff
@@ -35,22 +38,40 @@ public type Listener object {
...
     public function __attach(service s, string? name = ()) returns error? {
         return register(self, s, name);
     }
 
+    # Stops listening to the directory and detaches service from the `file:Listener`.
```
Comment (praneesha) [inline stdlib/file/src/main/ballerina/src/file/service_endpoint.bal:71]:
```suggestion
    # Stops listening to the directory and detaches the service from the `file:Listener`.
```

File: stdlib/file/src/main/ballerina/src/file/service_endpoint.bal [hunk 46]
```diff
@@ -35,22 +38,40 @@ public type Listener object {
...
+    # 
+    # + s - Type descriptor of the service
+    # + return - (), or else error upon failure to detach to the service
```
Comment (praneesha) [inline stdlib/file/src/main/ballerina/src/file/service_endpoint.bal:74]:
```suggestion
    # + return - () or else error upon failure to detach to the service
```
Do we need to specify the error?

File: stdlib/file/src/main/ballerina/src/file/file.bal (hunk 13 above)
Comment (niveathika) [reply stdlib/file/src/main/ballerina/src/file/file.bal]: The encompassing Error object is named as `Error`, I've added the module name to be more clear. Added in dba6e94

File: stdlib/file/src/main/ballerina/src/file/file.bal (hunk 18 above)
Comment (niveathika) [reply stdlib/file/src/main/ballerina/src/file/file.bal]: The encompassing Error object is named as `Error`, I've added the module name to be more clear. Added in dba6e94

File: stdlib/file/src/main/ballerina/src/file/file.bal (hunk 33 above)
Comment (niveathika) [reply stdlib/file/src/main/ballerina/src/file/file.bal]: The encompassing Error object is named as `Error`, I've added the module name to be more clear. Added in dba6e94

**Pull request merged**